### PR TITLE
Service

### PR DESCRIPTION
**Pull request by selleim**

### CODE REVIEW

File: Dixit_Client/Model/Model.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+        private void InitService()
+        {
+            servicecallback = new DixitServiceCallback();
+            serviceclient = new DixitServiceClient(new InstanceContext(servicecallback));
+        }
+
+        public void Login(string username)
+        {
+            if (isloggedin) { return; }
```
Comment (VSZM) [inline Dixit_Client/Model/Model.cs:33]: Am I right that we allow "    \t" as a valid username here?

File: Dixit_Service/DixitService.svc.cs [hunk 2]
```diff
@@ -12,52 +12,173 @@
...
+        #endregion
         public CreateGameResult CreateGame(string name)
         {
-            throw new NotImplementedException();
+            var r = new CreateGameResult();
+            if (!Games.Any(x => x.GameName == name))
+            {
+                var gi = new GameInfo();
+                gi.GameName = name;
+                gi.Game = (null as IDixitGame); //TODO
```
Comment (VSZM) [inline Dixit_Service/DixitService.svc.cs]: https://github.com/VSZM/Dixit_By_MonkeyzWithKeyboardz_ELTE/blob/master/Dixit_Logic/Classes/DixitGame.cs

File: Dixit_Service/packages.config [hunk 3]
```diff
@@ -8,4 +8,5 @@
   <package id="Microsoft.ApplicationInsights.Web" version="1.2.3" targetFramework="net461" />
   <package id="Microsoft.ApplicationInsights.WindowsServer" version="1.2.3" targetFramework="net461" />
   <package id="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" version="1.2.3" targetFramework="net461" />
+  <package id="SimpleInjector" version="3.2.3" targetFramework="net461" />
```
Comment (VSZM) [inline Dixit_Service/packages.config]: Where are we using DI?

File: Dixit.Common/Injector.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿using SimpleInjector;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dixit.Common
+{
+    public static class Injector
```
Comment (VSZM) [inline Dixit.Common/Injector.cs:10]: Why not use a Dependency Injection framework instead?

**Review by VSZM — APPROVED** [review]

File: Dixit_Client/View/LoginWindow.xaml.cs [hunk 5]
```diff
@@ -22,6 +22,8 @@ public partial class LoginWindow : Window
         public LoginWindow()
         {
             InitializeComponent();
+            var model = new Dixit_Client.Model.Model();
+            model.Login("Asda");
```
Comment (SilverGeri) [inline Dixit_Client/View/LoginWindow.xaml.cs]: I don't think you need it here, loginwindow has its own viewmodel, you can create the model instance there

**Pull request merged**